subsys/bluetooth/services/fast_pair/fp_crypto/Kconfig.fp_crypto

```diff
@@ -36,6 +36,13 @@ config BT_FAST_PAIR_CRYPTO_MBEDTLS
 	help
 	  Select MbedTLS cryptographic backend for Fast Pair.
 
+config BT_FAST_PAIR_CRYPTO_OBERON
+	bool "Fast Pair with Oberon cryptographic backend"
+	select OBERON_BACKEND
```

Comment:
Shouldn't we select `NRF_OBERON` instead of these 2? You actually directly use API of the oberon library.

Reply:
Then you also need to manually link the library in CMakeLists.txt of fp_crypto if oberon is used:
`target_link_libraries(fp_crypto PRIVATE nrfxlib_crypto)`
That way of oberon library integration is suggested by library's Kconfig help.

Reply:
It also saves you quite a lot of memory, because you do not need to enable most of the features that you don't use :)

Reply:
Sure, seems like a good idea